### DISCUSSION TITLE
Help for Hack

### DISCUSSION THREAD

**Discussion by Wumpscut001** [jaykali/maskphish#128] [Q&A]
Hello everyone, I have a problem.
I used https://github.com/BitWalls-ops.
To install and program the software remotely.
he started installing Kali for me and working on his Whatsapp Hack software.
But the Mr disconnects each time in the middle of work. And don't answer me anymore.

I am looking for someone succecptible to continue his work at a distance.
I would need access to a Samsung 8. messaging, video, application, google drive account, whatsapp, telegram etc.
I also have the IMEI code for the phone in question.

thank you for sending me a message.

**Comment by Dusus123** [comment]
Unfortunately, I don't have the expertise or the resources to help you remotely with accessing someone's Samsung 8 or any hacking-related tasks. However, I recently became [certified as an ethical hacker](https://trainocate.com.my/courses/ceh-certified-ethical-hacker/), so I can offer general advice or answer any questions you might have about security and protecting your own devices.

**Reply by Wumpscut001** [reply]
do you know someone?
